Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/ViewSrg.azsli

```diff
@@ -12,12 +12,12 @@
 
 partial ShaderResourceGroup ViewSrg
 {
-    row_major float4x4 m_viewProjectionMatrix;
-    row_major float4x4 m_viewMatrix;
-    row_major float4x4 m_viewMatrixInverse;
-    row_major float4x4 m_projectionMatrix;
-    row_major float4x4 m_projectionMatrixInverse;
-    row_major float4x4 m_viewProjectionInverseMatrix;
+    row_major float4x4 m_viewProjectionMatrix;          // World -> Screen Projection
```

Comment:
Maybe `World -> Clip` is more accurate (I don't know why but "screen" to me has connotations of being scaled by screen extent or strictly screenspace uvs whereas clip space coordinates still possess depth)

Reply:
Good suggestion, will  change